demo/src/main/AndroidManifest.xml

```diff
@@ -20,6 +21,7 @@
         <activity
             android:name="com.braintreepayments.browserswitch.BrowserSwitchActivity"
             android:launchMode="singleTask"
+            android:exported="true"
```

Comment:
random q: does Android 12 require this?

Reply:
Yeah this is part of [Android 12 security updates](https://developer.android.com/about/versions/12/behavior-changes-12#exported)

Reply:
Wow yeah that's legit. We may need to update the docs a bit.